### PR TITLE
Restyle and update entire Guacamole website.

### PR DESCRIPTION
**Pull request by mike-jumper**
This change massively restyles the entire website, attempting to refresh the overall look. The home page has been restyled with curves that are reminiscent of the classic bowl logo:

Restyled Guacamole website home page

The release archives has been restyled to resemble a timeline:

Guacamole release archive

Other changes include:

* Switching over to "Inter", "Rubik", and "Inconsolata" as the fonts used throughout the site.
* Migrating to SVG images wherever possible.
* Migrating to SCSS for maintainability of styles.
* Creating new images for the feature list on the home page.
* Adding content to clarify how mailing lists are used, directly point to the code of conduct (and highlight that our mailing lists are not to be used to promote commercial products).
* Providing instructions for support companies interested in being listed.
* Cleaning up the structure of the navigation menu, adding duplicate entries as necessary to make it easier to find the mailing lists, bug tracker, etc.

### CODE REVIEW

**Comment by jmuehlner** [comment]
I still need to to go through this with a finer-toothed comb, but it's looking really good so far!

**Review by jmuehlner — APPROVED** [review]
Looks good to me! It'd probably be good to get another set of eyes on a change this big, though.

@necouchman what do you think?

**Comment by necouchman** [comment]
> Looks good to me! It'd probably be good to get another set of eyes on a change this big, though.
> 
> @necouchman what do you think?

Sure, I'll take a look through it.

File: _includes/footer.html [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-        Version 2.0</a>. The HTML5 shield logo has been modified from the
-        original work of the <a
-            href="http://www.w3.org/">World Wide Web Constortium (W3C)</a>,
-        which is licensed under the <a
-            href="http://creativecommons.org/licenses/by/3.0/">Creative
-        Commons Attribution 3.0</a> license.
-    </p>
+        <!-- Copyrights -->
+        <p class="copyright">
+            Copyright &copy; 2020 <a href="http://www.apache.org/">The Apache
```
Comment (necouchman) [inline _includes/footer.html:7]: Is 2020 still the correct year, or should this be pulled forward to 2024?

**Review by necouchman — COMMENTED** [review]
Just one item that I'm wondering if it should be updated - other than that I'm good with the changes.

File: _includes/footer.html (hunk 1 above)
Comment (mike-jumper) [reply _includes/footer.html:7]: It should be updated. I'll make that change.

Comment (mike-jumper) [reply _includes/footer.html]: Updated to dynamically fill the current year at build time.

**Review by necouchman — APPROVED** [review]

**Pull request merged**